AWSPluginsCore.podspec

```diff
@@ -10,7 +10,7 @@
 $AMPLIFY_VERSION = '1.8.1'
```

Comment:
Is the amplify version changing too with the release of this change? I'm a little unclear on the actual root cause here, why a patch version on the SDK breaking consumers of Amplify?